inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -222,6 +236,8 @@ static void showUsage() {
     std::cout << "    -progress                 " << progress_message << std::endl;
     std::cout << "    -shape                    " << shape_message << std::endl;
     std::cout << "    -layout                   " << layout_message << std::endl;
+    std::cout << "    -cache \"<path>\"           " << cache_dir_message << std::endl;
+    std::cout << "    -single_load              " << single_load_message << std::endl;
```

Comment:
Common comments:

1. Usual practice include python benchmark_app update as well. We keep these 2 tools aligned.
2. I think it is necessary to add test in sample smoke tests with new parameters.